docs/install/index.rst

```diff
@@ -40,30 +40,33 @@ install it with e.g. ``apt-get`` or ``yum``.
 Development version
 -------------------
 
-To install the development version of Gammapy::
+You can install the development version of Gammapy like this::
```

Comment:
Maybe mention this is the recommended way for a conda environment as well?

Reply:
@adonath - I think for the install and conda install we'll need follow-up PRs to improve what we have.
(note that there's still quite a bit of content in http://docs.gammapy.org/en/latest/notebooks/tutorial_setup.html)